app/controllers/api/base_controller.rb

```diff
@@ -39,8 +39,8 @@ class BaseController < ActionController::Base
 
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
-    def get_resource
-      instance_variable_get(:"@#{resource_name}") || raise('no resource loaded')
+    def get_resource(options = {})
```

Comment:
I would appreciate if Ruby 2.0 could get supported as long as possible, but if support for it gets dropped in the core code, there's of course nothing I can do except stopping trusty builds ;)